microprofile.jdt/pom.xml

```diff
@@ -14,7 +14,7 @@
     <tycho.extras.version>${tycho.version}</tycho.extras.version>
     <tycho.scmUrl>scm:git:https://github.com/eclipse/lsp4mp</tycho.scmUrl>
     <tycho.generateSourceReferences>true</tycho.generateSourceReferences>
-    <jdt.ls.version>0.57.0.20200619102332</jdt.ls.version>
```

Comment:
We might want to move this part to a different PR. What was the rational for using 0.57.0?